src/linear_algebra/nonsingular_inverse.lean

```diff
@@ -274,6 +277,11 @@ calc adjugate A ⬝ A = (Aᵀ ⬝ (adjugate Aᵀ))ᵀ :
   by rw [←adjugate_transpose, ←transpose_mul, transpose_transpose]
 ... = A.det • 1 : by rw [mul_adjugate (Aᵀ), det_transpose, transpose_smul, transpose_one]
 
+lemma det_adjugate_eq_one {A : matrix n n α} (h : A.det = 1) : (adjugate A).det = 1 :=
```

Comment:
How hard would it be to show the general formula `det (adjugate A) = det(A)^{n.card - 1}` (assuming, maybe `n.card ≥ 1`)?

Reply:
I managed to prove the general formula for the case where `det A` is invertible. The case `det A = 0` requires showing for all `B` that `det B = 0` iff there is a nonzero `v` such that `mul_vec B v = 0`, which seems to be quite a lot of work.

Reply:
Hmm, I'm not even sure if the general formula is true over a general ring... so assuming `det A` invertible is fine for now. It can always be refactored and generalised (-;